content/en/docs/Policy Reports/_index.md

```diff
@@ -17,9 +17,15 @@ Kyverno uses the policy report schema published by the [Kubernetes Policy WG](ht
 Policy reports show policy results for current resources in the cluster. For information on resources that were blocked during admission controls, use the [policy rule execution metric](/docs/monitoring-kyverno-with-prometheus-metrics/policy-rule-results-info/).
 {{% /alert %}}
 
-{{% alert title="Note" color="info" %}}
-Policy reports are available in Kyverno 1.3.0+. For older (1.2.x) versions, you can use `kubectl get polv -A` to view policy violations.
-{{% /alert %}}
+## Report result logic
+
+Entries in a policy report contain a `result` field which can be either `pass`, `skip`, `warn`, `error`, or `fail`.
+
+`pass`: The resource was applicable to a rule and the pattern passed evaluation.
+`skip`: Preconditions were not satisfied (if applicable) in a rule and so further processing was not performed. Non-resolved variables are substituted with `""` to allow existence checks to succeed.
```

Comment:
>Non-resolved variables are substituted with `""` to allow existence checks to succeed.

This is true for all status, should we move it out of `skip`?

Reply:
Can do.